pkgs/applications/video/mplayer/svn-r38188-ffmpeg44fix.patch

```diff
@@ -0,0 +1,22 @@
+Index: libmpcodecs/ad_spdif.c
+===================================================================
+diff --git a/libmpcodecs/ad_spdif.c b/libmpcodecs/ad_spdif.c
+--- a/libmpcodecs/ad_spdif.c	(revision 38198)
++++ b/libmpcodecs/ad_spdif.c	(revision 38199)
+@@ -298,14 +298,8 @@
+         if (spdif_ctx->header_written)
+             av_write_trailer(lavf_ctx);
+         av_freep(&lavf_ctx->pb);
```

Comment:
I find it a bit suspicious that this line remained. (On a quick glance; I don't know the APIs.)

Reply:
You're right, it shouldn't be freed like that, but that's hardly the worst thing that's wrong with this file. It should be `avformat_close_input()`, but that's something that upstream needs to fix.